Automattic/jetpack
# Widget Visibility: add missing jetpack Config external

## Proposed changes:

This should avoid build warnings like this:

[plugins/jetpack 1:16.306] [build-production-widget-visibility] Module not found: Error: Can't resolve 'jetpackConfig' in '/home/runner/work/jetpack/jetpack/projects/js-packages/config/src'

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Start with the connected Jetpack plugin, on a site using a non-block theme like Twenty Twenty One.
* Go to Jetpack > settings, and enable the Widget Visibility module
* Go to Appearance > Widgets
* Add a block to a widget area.
* In the block's settings on the right, scroll down to the widget visibility area and use it; it should work just fine.
